src/smart-components/Thread/components/ParentMessageInfo/index.scss

```diff
@@ -0,0 +1,123 @@
+@import '../../../../styles/variables';
+
+.sendbird-parent-message-info {
+  position: relative;
+  width: 100%;
+  height: fit-content;
+
+  padding: 12px 12px 12px 16px;
+  display: inline-flex;
+  flex-direction: row;
+  align-items: flex-start;
+  box-sizing: border-box;
+}
+
+// left
+.sendbird-parent-message-info__sender {
+  position: relative;
+  min-width: 40px;
+  min-height: 40px;
+}
+
+// right
+.sendbird-parent-message-info__content {
+  position: relative;
+  margin-left: 12px;
+
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.sendbird-parent-message-info__content__info {
+  position: relative;
+  max-width: 188px;
+  height: 16px;
+
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+.sendbird-parent-message-info__content__info__sender-name {
+  position: relative;
+  max-width: 142px;
+
+  word-break: keep-all;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
we have so many ellipsis, might consider making mixin https://gist.github.com/Olivier-Go/18b376150f764e75a6e78bc546fb0b4f

Reply:
This is really good idea, let's apply it in Dec